lib/osf-components/addon/components/file-browser/file-item/template.hbs

```diff
@@ -57,8 +57,15 @@
                 </span>
             {{/if}}
             {{#unless @manager.selectedFiles}}
-                <FileActionsMenu @item={{@item}} @onDelete={{@manager.reload}} @manager={{@manager}} />
+                <FileActionsMenu @item={{@item}} @onDelete={{@manager.reload}} @manager={{@manager}} @toggleRenameModal={{this.openRenameModal}} local-class='FileActionsButton'/>
+                <FileBrowser::FileRenameModal
+                    @isOpen={{this.renameModalOpen}}
+                    @onClose={{action this.closeRenameModal}}
+                    @item={{@item}}
+                    @manager={{@manager}}
+                    @disabled={{this.disabled}}
+                />
```

Comment:
Plz move the invocation of <FileBrowser::FileRenameModal>` within `<FileActionsMenu>`.